GIT_AND_GITHUB.md

```diff
@@ -57,23 +57,11 @@ Besides, a description with more information could be helpful, for example:
 - If it's an interaction, a gif would help a lot. A good tool to make gifs is [Licecap](http://www.cockos.com/licecap/).
 - If it's a bug, steps to reproduce are very useful to help understanding context.
 
-#### When to merge a pull request?
-
 Before merging a pull request, your code has to be reviewed, so that we can learn from you, 
 point out your silly mistakes, and/or post a sufficient amount of gifs. The reviewing
```

Comment:
`point out the silly mistakes we all make once in a while`? seems friendlier and less condemning 😄